plugins/sagemaker/setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import setup
+
+PLUGIN_NAME = "sagemaker"
```

Comment:
what about putting all aws related plugins in one place? this the plugin is `aws` and we can have `aws.sagemaker` as the first thing in there?

Reply:
Yeah sure, but I think this will get pretty big.  In the future we should do `aws[sagemaker]`?  but for now install everything.

Reply:
i guess you are right, just keep sagemaker separate - then?